<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenControl</class>
 <widget class="QMainWindow" name="OpenControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenControl</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>//media/Eli/Projects/OpenControl/Python Source/gui/OpenComputers.ico</normaloff>//media/Eli/Projects/OpenControl/Python Source/gui/OpenComputers.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="1" column="0">
     <widget class="QGroupBox" name="inputPower">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="title">
       <string>Input Power</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="PlotWidget" name="inputGraph"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="totalPower">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Total Power</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="PlotWidget" name="totalGraph"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="applied_energistics">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="title">
       <string>Applied Energistics 2</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" alignment="Qt::AlignVCenter">
        <widget class="QPushButton" name="storage">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Storage</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" alignment="Qt::AlignVCenter">
        <widget class="QPushButton" name="craftingRequest">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Crafting Request</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionConnect_All"/>
    <addaction name="separator"/>
    <addaction name="actionConnect_to_Game"/>
    <addaction name="actionConnect_To_Database"/>
    <addaction name="actionConnect_to_Relay_Server"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuConnection_Settings">
     <property name="title">
      <string>Connection Settings</string>
     </property>
     <widget class="QMenu" name="menuDatabase">
      <property name="title">
       <string>Database</string>
      </property>
      <addaction name="actionName"/>
     </widget>
     <addaction name="actionIP"/>
     <addaction name="actionLogin_Credentials"/>
     <addaction name="menuDatabase"/>
     <addaction name="actionAuto_Connect"/>
    </widget>
    <addaction name="menuConnection_Settings"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect_All">
   <property name="text">
    <string>Connect All</string>
   </property>
  </action>
  <action name="actionConnect_to_Game">
   <property name="text">
    <string>Connect to Game</string>
   </property>
  </action>
  <action name="actionConnect_To_Database">
   <property name="text">
    <string>Connect To Database</string>
   </property>
  </action>
  <action name="actionConnect_to_Relay_Server">
   <property name="text">
    <string>Connect to Relay Server</string>
   </property>
  </action>
  <action name="actionIP">
   <property name="text">
    <string>IP/Port</string>
   </property>
  </action>
  <action name="actionLogin_Credentials">
   <property name="text">
    <string>Login Credentials</string>
   </property>
  </action>
  <action name="actionAuto_Connect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Connect</string>
   </property>
  </action>
  <action name="actionName">
   <property name="text">
    <string>Name</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
